bom/pom.xml

```diff
@@ -141,6 +142,16 @@
                 <artifactId>sofa-rpc-all</artifactId>
                 <version>${sofa.rpc.version}</version>
             </dependency>
+            <dependency>
```

Comment:
just dependencyManagement in bom.pom and not contains in all.pom, must be dependency separately when the user uses it.

Reply:
The default serialization is `fastjson` now, so I think if the users want to switch to `protostuff`, they need to add additional dependencies by themselves. But if the default is `protostuff`,  we can add dependencies of `protostuff` by default.    

About this,  @slievrly  @leizhiyuan  Do you have any good opinions?

Reply:
I think we can add by default.

Reply:
I think the solution is the same as the existing multiple registration centers and configuration centers.